src/components/ConnectButton/index.jsx

```diff
@@ -1,76 +1,78 @@
-import Torus from '@toruslabs/torus-embed'
-import WalletConnectProvider from '@walletconnect/web3-provider'
-import Fortmatic from 'fortmatic'
-import Portis from '@portis/web3'
+import Onboard from 'bnc-onboard'
 import Button from '~/components/layout/Button'
 import { fetchProvider, removeProvider } from '~/logic/wallets/store/actions'
-import { getNetwork } from '~/config'
+import { getNetworkId } from '~/config'
 import { store } from '~/store'
 
 const isMainnet = process.env.REACT_APP_NETWORK === 'mainnet'
 
-const PORTIS_DAPP_ID = isMainnet ? process.env.REACT_APP_PORTIS_ID : '852b763d-f28b-4463-80cb-846d7ec5806b'
+const BLOCKNATIVE_API_KEY = isMainnet
+  ? process.env.REACT_APP_BLOCKNATIVE_KEY
+  : '7fbb9cee-7e97-4436-8770-8b29a9a8814c'
+const PORTIS_DAPP_ID = isMainnet
+  ? process.env.REACT_APP_PORTIS_ID
+  : '852b763d-f28b-4463-80cb-846d7ec5806b'
 // const SQUARELINK_CLIENT_ID = isMainnet ? process.env.REACT_APP_SQUARELINK_ID : '46ce08fe50913cfa1b78'
-const FORTMATIC_API_KEY = isMainnet ? process.env.REACT_APP_FORTMATIC_KEY : 'pk_test_CAD437AA29BE0A40'
+const FORTMATIC_API_KEY = isMainnet
+  ? process.env.REACT_APP_FORTMATIC_KEY
+  : 'pk_test_CAD437AA29BE0A40'
 
-export const web3Connect = new Web3Connect.Core({
-  network: getNetwork().toLowerCase(),
-  providerOptions: {
-    walletconnect: {
-      package: WalletConnectProvider,
-      options: {
-        infuraId: process.env.REACT_APP_INFURA_TOKEN,
-      },
-    },
-    portis: {
-      package: Portis,
-      options: {
-        id: PORTIS_DAPP_ID,
-      },
-    },
-    fortmatic: {
-      package: Fortmatic,
-      options: {
-        key: FORTMATIC_API_KEY,
-      },
-    },
-    torus: {
-      package: Torus,
-      options: {
-        enableLogging: false,
-        buttonPosition: 'bottom-left',
-        buildEnv: process.env.NODE_ENV,
-        showTorusButton: true,
-      },
-    },
+const wallets = [
+  { walletName: 'coinbase' },
+  { walletName: 'metamask', preferred: true },
+  {
+    walletName: 'walletConnect',
+    preferred: true,
+    infuraKey: process.env.REACT_APP_INFURA_TOKEN,
   },
-})
+  { walletName: 'trust', preferred: true },
+  { walletName: 'dapper', preferred: true },
+  {
+    walletName: 'fortmatic',
+    apiKey: FORTMATIC_API_KEY,
+    preferred: true,
+  },
+  {
+    walletName: 'portis',
+    apiKey: PORTIS_DAPP_ID,
+    preferred: true,
+    label: 'Login with Email',
+  },
+  { walletName: 'authereum' },
+  { walletName: 'opera' },
+  { walletName: 'operaTouch' },
+]
 
-web3Connect.on('connect', (provider: any) => {
-  if (provider) {
-    store.dispatch(fetchProvider(provider))
-  }
+export const onboard = new Onboard({
+  dappId: BLOCKNATIVE_API_KEY,
+  networkId: getNetworkId(),
+  subscriptions: {
+    wallet: (wallet) => store.dispatch(fetchProvider(wallet.provider)),
+  },
+  walletSelect: {
+    wallets,
+  },
 })
 
-web3Connect.on('disconnect', () => {
-  store.dispatch(removeProvider())
-})
+// web3Connect.on('disconnect', () => {
```

Comment:
should this be removed?

Reply:
yes, this is still WIP. I just create the PR to get the autodeployment so I can share it within the team to test the flow

Reply:
I think you could use "Draft" prs (https://github.blog/2019-02-14-introducing-draft-pull-requests/) for that ;) (if deployment is triggered for them) :)

Reply:
lol I noticed this feature is used by Protofire team, but I couldn't find how I can do this 😄 thanks